resources/sdk/purecloudjava/templates/README.mustache

```diff
@@ -128,6 +128,25 @@ Specify the connector in the builder:
 .withProperty(ApiClientConnectorProperty.CONNECTOR_PROVIDER, new OkHttpClientConnectorProvider())
 ```
 
+#### Setting the max retry time
+
+In default, Java SDK makes a call to API without a Retry option but this feature is configurable.
```

Comment:
This paragraph could use some rewording:

> By default, the Java SDK does not automatically retry any failed requests. To enable automatic retries, provide a `RetryConfiguration` object with the maximum number of seconds to retry requests when building the `ApiClient` instance.

Reply:
Updated the docs.